pkg/generators/cluster.go

```diff
@@ -65,9 +64,12 @@ func (g *ClusterGenerator) GenerateParams(
 
 	res := make([]map[string]string, len(clusterSecretList.Items))
 	for i, cluster := range clusterSecretList.Items {
-		params := make(map[string]string, len(cluster.ObjectMeta.Labels) + 2)
+		params := make(map[string]string, len(cluster.ObjectMeta.Labels)+2)
```

Comment:
the length of the map should be: len(cluster.ObjectMeta.Labels) + len(cluster.ObjectMeta.Annotations) + 2

Reply:
Fixed. Thanks!